remediation_worker/jobs/azure_key_vault_logging_for_keyvault_enabled/README.md

```diff
@@ -38,13 +41,16 @@ You may run test using following command under vss-remediation-worker-job-code-p
     python3 -m pytest test
 ```
 ## Deployment
-Provision a Virtual Machine Create an EC2 instance to use for the worker. The minimum required specifications are 128 MB memory and 1/2 Core CPU.
-Setup Docker Install Docker on the newly provisioned EC2 instance. You can refer to the [docs here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html) for more information.
-Deploy the worker image SSH into the EC2 instance and run the command below to deploy the worker image:
+Provision a Virtual Machine Create an Azure Virtual Machine instance to use for the worker. The minimum required specifications are 128 MB memory and 1/2 Core CPU.
+Setup Docker Install Docker on the newly provisioned Azure Virtual Machine instance. You can refer to the [docs here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html) for more information.
+Deploy the worker image SSH into the Azure Virtual Machine instance and run the command below to deploy the worker image:
```

Comment:
Deploy the worker docker image by SSH into the Azure Virtual Machine instance and run the following commands.